src/main/webapp/WEB-INF/views/car/addDriver.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+  <title>Add driver to car</title>
+</head>
+<body>
+<h3>Write down values for add driver to car:</h3>
+<form method="post" action="${pageContext.request.contextPath}/cars/drivers/add">
+  Driver id <input type="number" name="driver_id"><br>
+  Car id <input type="number" name="car_id"><br>
+  <button type="submit">Submit</button>
+</form>
```

Comment:
<img width="971" alt="image" src="https://user-images.githubusercontent.com/58032907/203939335-960b07b9-b14d-4b40-9777-89228df57852.png">
should be `required` and of type `number`